### ISSUE TITLE
3.0.8 is It puts a ### sign to url_config and does not save any record. there is bug

### ISSUE THREAD

**Issue by strongerr1234** [ihmily/DouyinLiveRecorder#678]
### ⚠️ 确认是否已存在类似问题

- [X] 我已经搜索过issues，没有找到类似问题

### 🔧 运行方式

直接运行的exe文件

### 🐍 如果是使用源代码运行，请选择你的Python环境版本

Python 3.8

### ⚠️ 确认是否已经重试多次

- [X] 我已经尝试过多次，仍然出现问题

### 🕹 复现步骤

3.0.8 is It puts a ### sign to url_config and does not save any record. there is bug

### 😯 问题描述

3.0.8 is It puts a ### sign to url_config and does not save any record. there is bug

### 📜 错误信息

3.0.8 is It puts a ### sign to url_config and does not save any record. there is bug

**Comment by ihmily** [comment]
are there any other errors ? it just need one # to stop the recording

**Comment by strongerr1234** [comment]
> are there any other errors ? it just need one # to stop the recording

 dont know if there any bugs cause i cant recording anything so bug is take a ### sign to url_config for my all url links. its interesting.

**Comment by ihmily** [comment]
please remove # manually，and check  for duplicate links，only keep the one link.

**Comment by strongerr1234** [comment]
> please remove # manually，and check for duplicate links，only keep the one link.

ohh no. i mean example: i have too much url link ready in url_config.ini. so there 700 url link for record but when i put my reay of url_config file to 3.0.8 version then all url link first line getting "###" sign and not recording :(

**Comment by ihmily** [comment]
Yes, I caught the bug, but it only appears when entering an unsupported live streaming links. 
When input unknown link, the program will auto add one "#" to the beginning of the link, because it has a bug, will Infinite loop to add "#", I will fix it.

But "#" will not be actively added before the supported live link, if "#" is added to all link, then you need to check and re-enter these links yourself.

**Comment by strongerr1234** [comment]
> Yes, I caught the bug, but it only appears when entering an unsupported live streaming links. When input unknown link, the program will auto add one "#" to the beginning of the link, because it has a bug, will Infinite loop to add "#", I will fix it.
> 
> But "#" will not be actively added before the supported live link, if "#" is added to all link, then you need to check and re-enter these links yourself.

ohh no no. so i have supported links already and i doing with that in 3.0.7 and that links are tiktok and bigo live only. so just happening in 3.0.8 and getting not only "#" so there three sign like this "###" and not recording. so 3.0.8 breaking my all url links in url_config with first line at "###" sign
